prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -181,7 +184,7 @@ public async Task<ActionResult> UpdateEnrolleePaperSubmissionSelfDeclarations(in
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
         [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        public async Task<ActionResult> UpdateEnrolleePaperSubmissionDocuments(int enrolleeId, IEnumerable<Guid> payload)
+        public async Task<ActionResult> UpdateEnrolleePaperSubmissionDocuments(int enrolleeId, IEnumerable<PaperEnrolleeDocumentViewModel> payload) //IEnumerable<Guid> payload)
```

Comment:
Some leftover dev code with the comments here and the two unused `using`

Reply:
Deleted